cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/ClusterBootstrapSettings.scala

```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
  */
-package akka.cluster.bootstrap
+package akka.management.cluster.bootstrap
```

Comment:
all tools here are now in management, except the discovery -- that's `akka.discovery`